src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -472,15 +472,11 @@ private void handleCloudWatchAttemptStatus(CloudWatchAttempt cloudWatchAttempt)
                 return;
             }
             completedFiles.forEach(file -> {
-                try {
-                    boolean successfullyDeleted = Files.deleteIfExists(Paths.get(file.getSourcePath()));
-                    if (successfullyDeleted) {
-                        logger.atDebug().log("Successfully deleted file with name {}", file.getSourcePath());
-                    } else {
-                        logger.atWarn().log("Unable to delete file with name {}", file.getSourcePath());
-                    }
-                } catch (IOException e) {
-                    logger.atError().cause(e).log("Unable to delete file with name: {}", file.getSourcePath());
+                boolean successfullyDeleted = file.delete();
```

Comment:
Is this `file` referring to the hardlink one or the original one?

Reply:
Could be any. It is referring to a log file. If we rolled back to the old functionality it refers to a regular java file if not it refers to the hardlink

Reply:
Ignore above comment, just see the `delete` method you defined.